downloadSpecs.js

```diff
@@ -0,0 +1,37 @@
+const http = require('https');
+const fs = require('fs');
+
+const specsRepo = 'https://raw.githubusercontent.com/paritytech/smoldot/main/bin';
+const chainDir = '.chains';
+
+const networks = [
+  'polkadot', 'kusama', 'westend' // , 'rococo', 'tick', 'track', 'trick'
+];
+
+const paths = [
+  'packages/connect/assets',
+  'projects/extension/public/assets',
+]
+
+console.log(`Init downloading of specs for chains ${networks.join(', ')}.`);
+let promise = new Promise((resolve) => {
```

Comment:
It would be better to create a Promise for each file and await `Promise.all` to download in parallel.